scripts/build-multi-pwa.js

```diff
@@ -32,7 +33,7 @@ if (parallel) {
 
 const result = cp.spawnSync(
   path.join('node_modules', '.bin', 'npm-run-all' + (process.platform === 'win32' ? '.cmd' : '')),
-  ['ngcc', ...parallel, ...builds],
+  ['--silent', 'ngcc', ...parallel, ...builds],
```

Comment:
Yeah, I've found that output also pretty anoying but good for debuging purposes.